.github/workflows/load-test.yaml

```diff
@@ -106,25 +170,36 @@ jobs:
           command: |
             helm repo add openfga https://openfga.github.io/helm-charts
             helm repo update
-            helm upgrade --install openfgabench-${{ steps.vars.outputs.sha_short }} openfga/benchmark \
+            helm upgrade --install openfgabench-${{ needs.upload-code-to-ecr.outputs.sha_short  }} openfga/benchmark \
               --set Nparam=${{ github.event.inputs.N }} \
               --set Mparam=${{ github.event.inputs.M }} \
               --set Kparam=${{ github.event.inputs.K }} \
+              --set testListObject=true \
+              --set testCheck=true \
               --set testCase=${{ github.event.inputs.testCase }} \
               --set replicaCount=1 \
+              --set k6.projectID=${{ secrets.K6_PROJECT_ID}} \
               --set image.repository=${{ secrets.OPENFGA_ECR_URI }} \
-              --set image.tag=openfgabench-${{ steps.vars.outputs.sha_short }} \
+              --set image.tag=openfgabench-${{ needs.upload-code-to-ecr.outputs.sha_short }} \
               --set datastore.engine=postgres \
               --set datastore.uri="postgres://postgres:password@openfga-postgresql.default.svc.cluster.local:5432/postgres?sslmode=disable" \
               --set postgres.enabled=true \
               --set postgresql.auth.postgresPassword=password \
               --set postgresql.auth.database=postgres \
               --set postgresql.primary.resources.requests.memory=2Gi,postgresql.primary.resources.requests.cpu=1.0 \
-              --set openfga.resources.requests.memory=1Gi,openfga.resources.requests.cpu=500m
+              --set openfga.resources.requests.memory=1Gi,openfga.resources.requests.cpu=500m \
+              --set resources.requests.memory=1Gi,resources.limits.memory=2Gi
+
+      - name: wait
```

Comment:
I think this workflow job will timeout before the `wait` job won't it?

Reply:
🤔 no, the job will take as long as it needs (e.g. https://github.com/openfga/openfga/actions/runs/5205481848/jobs/9391000235). This particular step is capped at 1 hour.